### ISSUE TITLE
Help Needed

### ISSUE THREAD

**Issue by sarfraznawaz2005** [pratikborsadiya/vali-admin#33]
Thanks for the very nice admin package.

For my requirement, I need the links on top nav/header on the left hand side so I added them as can be seen in screenshot (links `Dashboard` and `Quick Links`):

![image](https://user-images.githubusercontent.com/201788/36063419-6d117e58-0e9e-11e8-8887-e9bce99d890b.png)

To add them, I simply added links under `<header class="app-header">` like so:

````
<header class="app-header">
    <a class="app-header__logo" href="{{route('admin_panel')}}">{{appName()}}</a>
    <a class="app-sidebar__toggle" href="#" data-toggle="sidebar"></a>

    <!-- Navbar Left -->
    <a class="app-nav__item {{active('admin_panel')}}" href="{{route('admin_panel')}}">Dashboard</a>

    <a class="app-nav__item dropdown-toggle" id="dropdownNav"
       data-toggle="dropdown"
       aria-haspopup="true"
       href="#"
       aria-expanded="false">
        Quick Links
    </a>

    <div class="dropdown-menu" aria-labelledby="dropdownNav">
        <a class="dropdown-item" href="#">Action</a>
        <a class="dropdown-item" href="#">Another action</a>
        <a class="dropdown-item" href="#">Something else here</a>
    </div>
````

All good but problem happens with responsive view:

![image](https://user-images.githubusercontent.com/201788/36063446-c4f6333e-0e9e-11e8-9032-427ededf3eb7.png)

As you can see, it sidebar and top nav is messed up on smaller screen. I am sure I am not adding links on left side of header/nav correctly.

Can you please tell me how do I add links on left side of header so that work fine on both desktop/mobile ?

In other words, we should be able to add links on both left and right side and it should work fine in responsive view/smaller screen when we click on little toggle button and should somehow also show all these links we have added.

Thanks for the help.

**Comment by pratikborsadiya** [comment]
Hi @sarfraznawaz2005 

The links on top navbar cannot be responsive, What you can do is use bootstrap utility classes mentioned [here](http://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements) to hide the elements in top navbar and show them in left sidebar on small screen.
